libs/SalesforceReact/package.json

```diff
@@ -8,31 +8,28 @@
   "dependencies": {
     "create-react-class": "^15.7.0",
     "react": "18.2.0",
-    "react-native": "0.72.7",
-    "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev"
+    "react-native": "0.73.6",
+    "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn73"
```

Comment:
Pointing to wmathurin rn73 to allow tests to pass. Will revert that line before merging.

Reply:
Locally I did not see the problem but I had not removed the node modules directory before running install :-(

Adding back with [101db25](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2532/commits/101db25204d24f6a295bb0631d6f8aa90d7e9c86)